packages/api/test/__fixtures__/sdk/operationid-quirks/index.ts

```diff
@@ -1,7 +1,7 @@
-import type { ConfigOptions, FetchResponse } from 'api/dist/core'
+import type { ConfigOptions, FetchResponse } from '@api/core'
 import Oas from 'oas';
-import APICore from 'api/dist/core';
-import definition from '../../../__fixtures__/definitions/operationid-quirks.json';
+import APICore from '@api/core';
```

Comment:
Moving core also lets us do this clean require instead of having to set this up in `exports`.